### PR TITLE
Rename Microsoft.Runtime.Abstractions to Microsoft.Extensions.PlatformAbstractions

### PR DESCRIPTION
**Pull request by pakrym**
@davidfowl @victorhurdugaci @moozzyk

### CODE REVIEW

**Comment by davidfowl** [comment]
What about the PRogram.Main

**Comment by pakrym** [comment]
Fixed in second commit

File: src/Microsoft.Dnx.Watcher/Program.cs [hunk 1]
```diff
@@ -17,17 +17,17 @@ public class Program
     {
         private const string DnxWatchArgumentSeparator = "--dnx-args";
 
-        private readonly ILoggerFactory _loggerFactory;
+        private static readonly ILoggerFactory _loggerFactory;
 
-        public Program(IRuntimeEnvironment runtimeEnvironment)
+        static Program()
```
Comment (victorhurdugaci) [inline src/Microsoft.Dnx.Watcher/Program.cs]: Change it back to normal constructor and create an instance of this in Main

File: src/Microsoft.Dnx.Watcher/Program.cs [hunk 2]
```diff
@@ -63,7 +63,7 @@ internal static void SeparateWatchArguments(string[] args, out string[] watchArg
             }
         }
 
-        private int MainInternal(string[] watchArgs, string[] dnxArgs, CancellationToken cancellationToken)
+        private static int MainInternal(string[] watchArgs, string[] dnxArgs, CancellationToken cancellationToken)
```
Comment (victorhurdugaci) [inline src/Microsoft.Dnx.Watcher/Program.cs]: Don't make it static

File: src/Microsoft.Dnx.Watcher/Program.cs (hunk 1 above)
Comment (pakrym) [inline src/Microsoft.Dnx.Watcher/Program.cs]: Done

**Comment by victorhurdugaci** [comment]
:shipit: after you rebase/squash

**Pull request merged**